README.md

```diff
@@ -42,6 +42,14 @@ module Pangraph.GraphML.Writer
 ...
 ```
 
+## Binary generation
+
+The command below should be used for the compilation and the generation of a binary file:
+
+```
+ghc -isrc -ipangraph pangraph/Main.hs -o Pangraph
```

Comment:
It may be better to provide a `stack setup/build` interface instead of always passing these parameters by hand.